pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -255,10 +255,10 @@ func (o *OpenIDCProvider) userToPrincipal(userInfo *oidc.UserInfo, info claimInf
 		displayName = userInfo.Email
```

Comment:
Nit: use arg variable name of `claiminfo` instead of `info`. As both those variables have info in their name.

Reply:
updated `info` to `claimInfo` to match the `userInfo` variable